micropython/mip/mip/__init__.py

```diff
@@ -123,12 +123,8 @@ def _install_json(package_json_url, index, target, version, mpy):
 
 
 def _install_package(package, index, target, version, mpy):
-    if (
-        package.startswith("http://")
-        or package.startswith("https://")
-        or package.startswith("github:")
-    ):
-        if package.endswith(".py") or package.endswith(".mpy"):
+    if package.startswith(("http://", "https://", "github:")):
+        if package.endswith((".py", ".mpy")):
```

Comment:
MicroPython doesn't support tuples of strings passed to startswith/endswith, so please revert this change.

Can ruff be configured to exclude this particular warning?